Makefile

```diff
@@ -1,11 +1,5 @@
 PROJECT_HOME_DIR := ${CURDIR}
 
-build:
-    # Builds a TensorFlow 2 ART docker container
-    # IMPORTANT ! If you have an existing python env folder make sure to first add it to the `.dockerIgnore` \
-    to reduce the size of your the art docker image
-	docker build -t project-art-tf2 .
-
 build1:
```

Comment:
The removal of the `build` target should be documented in the PR description. It is crucial to explain why the target is no longer needed.
Ensure that the deleted lines are not required by any other part of the build process.